external/nmea/parse.c

```diff
@@ -390,7 +390,7 @@ int nmea_parse_GPGSV( const char *buff, int buff_sz, nmeaGPGSV *pack )
     return 0;
   }
 
-  if ( type != 'P' && type != 'N' )
+  if ( type != 'P' && type != 'N' && type != 'L' && type != 'A' && type != 'B' && type != 'Q' )
```

Comment:
P, N, L, A, B and Q messages are allowed here, but only P, N, L, A and B are allowed at https://github.com/qgis/QGIS/blob/a6ef3155bd9a00f1fc255d0c4326fd777bee7535/src/core/gps/qgsnmeaconnection.cpp#L113
Is it intended or is it a typo?

Reply:
@agiudiceandrea With U-Blox ZED-F9P the $ GQGSV message is not in "GSV" message standard, it is written differently.
For now I wanted to exclude it, to investigate if it is a problem with the ZED-F9P or a problem in the configuration of my GNSS receiver.

Reply:
In this case, should it be excluded also from this line?

Reply:
You're right, it could be canceled. I left it in order not to completely exclude the QZSS satellites even if they are not used in the Italian territory.
It would be nice to have a match with another type of GNSS receiver with a working NMEA log in Australia or Japan !!